src/__tests__/App.test.js

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+
+// import { Provider } from 'react-redux'; // Import Provider from react-redux
+import '@testing-library/jest-dom/extend-expect'; // For better DOM assertions
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import App from '../App';
+
+test('renders the App component correctly', () => {
+  const { container } = render(<Provider store={store}><App /></Provider>);
+  expect(container).toMatchSnapshot();
+});
```

Comment:
So that you know, one of your tests is failing. kindly fix it 😉.
![Screenshot from 2023-08-04 02-34-18](https://github.com/henask12/space-travelers-hub/assets/44624138/6212c18b-e071-4c1f-aa9b-e669a73be64c)

Reply:
Hi @rotimiazeez 

we have fixed and resolved it like on the below screenshot - it should now work as expected
![image](https://github.com/henask12/space-travelers-hub/assets/37038066/430e508f-bf77-4bd7-8d24-e5a5cf5ba694)